diff --git a/hw/vendor/tagctrl/src/axi_tagctrl_reg_wrap.sv b/hw/vendor/tagctrl/src/axi_tagctrl_reg_wrap.sv
index 313372f..bbb33a2 100644
--- a/src/axi_tagctrl_reg_wrap.sv
+++ b/src/axi_tagctrl_reg_wrap.sv
@@ -295,4 +295,9 @@ module axi_tagctrl_reg_wrap #(
       .cached_end_addr_i
   );
 
+  // Assert Known for module outputs
+  `ASSERT_KNOWN(TaggedSubordinateResponseKnown, slv_resp_o)
+  `ASSERT_KNOWN(UntaggedManagerRequestKnown, mst_req_o)
+  `ASSERT_KNOWN(ConfigurationSubordinateResponseKnown, conf_resp_o)
+
 endmodule
